library/src/scripts/embeddedContent/Components.story.tsx

```diff
@@ -0,0 +1,110 @@
+/**
+ * @author Stéphane LaFlèche <stephane.l@vanillaforums.com>
+ * @copyright 2009-2019 Vanilla Forums Inc.
+ * @license GPL-2.0-only
+ */
+
+import { EmbedContainer, EmbedContainerSize } from "@library/embeddedContent/EmbedContainer";
+import { StoryHeading } from "@library/storybook/StoryHeading";
+import { storiesOf } from "@storybook/react";
+import React from "react";
+import { StoryTiles } from "@library/storybook/StoryTiles";
+import Button from "@library/forms/Button";
+import { StoryTile } from "@library/storybook/StoryTile";
+import { StoryParagraph } from "@library/storybook/StoryParagraph";
+
+const reactionsStory = storiesOf("Components", module);
+
+reactionsStory.add("Buttons", () => {
+    return (
+        <>
+            <StoryHeading depth={1}>Buttons</StoryHeading>
+            <StoryTiles>
+                <>
+                    <StoryTile>
+                        <>
+                            <Button>Standard</Button>
+                        </>
+                    </StoryTile>
+                    <StoryTile>
+                        <>
+                            <Button>Text</Button>
+                        </>
+                    </StoryTile>
+                    <StoryTile>
+                        <>
+                            <Button>Text Primary</Button>
+                        </>
+                    </StoryTile>
+                    <StoryTile>
+                        <>
+                            <Button>Icon</Button>
+                        </>
+                    </StoryTile>
+                    <StoryTile>
+                        <>
+                            <Button>Icon_Compact</Button>
+                        </>
+                    </StoryTile>
+                    <StoryTile>
+                        <>
+                            <Button>Compact</Button>
+                        </>
+                    </StoryTile>
+                    <StoryTile>
+                        <>
```

Comment:
Are all of these fragments necessary? What's the reason to have

```tsx
<StoryTile>
   <div></div>
   <span></span>
</StoryTile>

// Instead of 
<StoryTile>
   <>
     <div></div>
     <span></span>
   </>
</StoryTile>

```